packages/edit-site/src/components/global-styles/custom-css.js

```diff
@@ -64,6 +77,15 @@ function CustomCSSControl() {
 					</>
 				}
 			/>
+			{ cssError && (
+				<Notice
+					status="warning"
+					onRemove={ () => setCSSError( null ) }
+					politeness="polite"
```

Comment:
I'm not sure if this is acceptable TBH. Is it too disruptive?

Reply:
It might be a good idea to get some design feedback on this.

Recently we replaced all notices inside block placeholders with snackbars. Maybe we can also do the same here, Snackbar + red outline.

cc @jasmussen

Reply:
Thanks for the ping. I see in this branch a notice below the input:

<img width="312" alt="notice" src="https://user-images.githubusercontent.com/1204802/212916577-e3968c79-a62f-41d0-824b-ecc9ad54f00c.png">

Indeed, that notice becomes a bit noisy. I wonder if we can do what Codepen does, which is to show a small exclamation mark in the corner:

<img width="90" alt="Screenshot 2023-01-17 at 14 56 28" src="https://user-images.githubusercontent.com/1204802/212917395-9b1afe07-671d-4823-8835-3d223c11746b.png">

It could be something like this:

<img width="1856" alt="Global styles custom CSS" src="https://user-images.githubusercontent.com/1204802/212918019-b2714e5b-973a-40d3-945b-cc81056f1501.png">

Focusing or hovering this icon would show the error message.

If such an icon+tooltip isn't feasible, a snackbar could work, but in this case I think positional contextuality can be useful.

I realize that actually highlighting the problematic line is beyond the scope of this PR, but it would pair nicely.

Speaking of beyond the scope of this PR, it would be nice to put the "Read more about CSS" link into an ellipsis menu, and allow the CSS panel to flex all the way to the bottom, as in the original mockups.

Reply:
Thank you, Joen.

I like the idea. Syntax highlighting can potentially absorb the error notice role in the future.

>  Speaking of beyond the scope of this PR, it would be nice to put the "Read more about CSS" link into an ellipsis menu, and allow the CSS panel to flex all the way to the bottom, as in the original mockups.

We should probably create a tracking issue for the remaining items. I think @glendaviesnz is already working on the list.

Reply:
For now I updated #30142, let me know if that works. The original design still feels valid to me.

Reply:
Thank you, Joen!

Reply:
> We should probably create a tracking issue for the remaining items

@Mamaduka https://github.com/WordPress/gutenberg/issues/30142 has a list of outstanding tasks in the issue description